src/components/navigationLoader/NavigationContainer.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import * as className from 'classnames';
+
+export interface INavigationProps {
+  className: string;
+}
+
+export class NavigationContainer extends React.Component<INavigationProps, any> {
```

Comment:
Could be renamed to `Navigation`. Since there is not multiple navigation, I don't see why we should call this container.

Reply:
As we were talking, Navigation is already taken in React-Vapor.

Since it's a sidebar menu, a more meaningful name could be `SideMenu` instead of `NavigationContainer`.

Then other filenames could also be renamed to follow the rule of `~SideMenu[component]`, like `SideMenuLoadingHeader`, `SideMenuLoadingItem`, etc.

Reply:
If anyone wants to vote for a better name please go ahead.

Reply:
Even better than `SideMenu`, could be `SideNav`.

Since it's a popular naming practice in the community:
http://materializecss.com/side-nav.html
https://material.angular.io/components/sidenav/examples